src/Query/Mysql/SubstringIndex.php

```diff
@@ -2,37 +2,47 @@
 
 namespace DoctrineExtensions\Query\Mysql;
 
-use Doctrine\ORM\Query\AST\Functions\FunctionNode,
-    Doctrine\ORM\Query\Lexer;
+use  Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use  Doctrine\ORM\Query\Lexer;
+use  Doctrine\ORM\Query\Parser;
+use  Doctrine\ORM\Query\SqlWalker;
 
 /**
+ * Class SubstringIndex
+ * @package DoctrineExtensions\Query\Mysql
  * @author Vas N <phpvas@gmail.com>
  */
-
 class SubstringIndex extends FunctionNode
 {
     public $string = null;
     public $delimiter = null;
     public $count = null;
 
-    public function parse(\Doctrine\ORM\Query\Parser $parser)
+    /**
+     * @param \Doctrine\ORM\Query\Parser $parser
```

Comment:
I guess this PHPDoc is not required since the parameter is already strongly typed?